go/test/endtoend/cluster/topo_process.go

```diff
@@ -324,6 +319,9 @@ func (topo *TopoProcess) ManageTopoDir(command string, directory string) (err er
 	url := topo.VerifyURL + directory
 	payload := strings.NewReader(`{"dir":"true"}`)
 	if command == "mkdir" {
+		if *topoFlavor == "etcd2" { // No need to create the empty prefix key in v3
```

Comment:
Given the comment for `ManageTopoDir`, don't we only use this function for `etcd` anyway? And can we remove any callers that do `mkdir` then since that seem superfluous then?

Reply:
Yeah, there's conflicting signals here. It seems like this was at least intended to be used with multiple topo flavors, but then in certain places it assumes etcd2. I'd prefer to keep it generic so that we can leverage that.

Reply:
It's test code, so we can keep it like this but it seems surprising that a generic HTTP call would work across different flavors here really, so it's probably `etcd` specific anyway? I doubt we run these `endtoend` tests with anything but `etcd`?